tools/exports.js

```diff
@@ -1,15 +1,4 @@
-exports["Compressor"] = Compressor;
 exports["Dictionary"] = Dictionary;
-exports["JS_Parse_Error"] = JS_Parse_Error;
```

Comment:
`minify()` can still throw `JS_Parse_Error`, but the way to test for it, at least in `bin/uglifyjs`, is to check for `.name` instead of `instanceof`

https://github.com/mishoo/UglifyJS2/pull/1877/files#diff-e5511a595da8371c33a33346f3e22fd4L190